server/src/main/java/io/crate/expression/symbol/SymbolVisitor.java

```diff
@@ -42,10 +42,6 @@ public R visitDynamicReference(DynamicReference symbol, C context) {
         return visitSymbol(symbol, context);
     }
 
-    public R visitVoidReference(VoidReference symbol, C context) {
```

Comment:
Also, why not needed anymore?

Reply:
Reverted, same as https://github.com/crate/crate/pull/17133#discussion_r1935150098.

Reply:
Reverted, same as https://github.com/crate/crate/pull/17133#discussion_r1935150098.